src/models.jl

```diff
@@ -48,6 +49,19 @@ mutable struct FitInput{T}
     end
 end
 
+"""
+    has_exogenous(fit_input::FitInput{T}) where T
+
+Returns true if fit_input has one or more exogenous time series and false otherwise.
+"""
+function has_exogenous(fit_input::FitInput{T}) where T
+    if length(fit_input.exogenous) >= 1
+        return true
+    else
+        return false
+    end
```

Comment:
`return length(fit_input.exogenous) >= 1`

Reply:
ok, ive just corrected and pushed again